include/event_manager/profiler.h

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_EVENT_MANAGER_PROFILER_H_
+#define ZEPHYR_INCLUDE_EVENT_MANAGER_PROFILER_H_
+
+/**
+ * @defgroup profiler Profiler
+ * @brief Profiler
+ *
+ * @{
+ */
+
+
+#include <zephyr/types.h>
+#include <sys/util.h>
+#include <sys/__assert.h>
+
+#ifndef CONFIG_MAX_NUMBER_OF_CUSTOM_EVENTS
+/** Maximum number of custom events. */
+#define CONFIG_MAX_NUMBER_OF_CUSTOM_EVENTS 0
+#endif
+
+/** @brief Set of flags for enabling/disabling profiling for given event types.
+ */
+extern uint32_t profiler_enabled_events;
+
+
+/** @brief Number of event types registered in the Profiler.
+ */
+extern uint8_t profiler_num_events;
+
+
+/** @brief Data types for profiling.
+ */
+enum profiler_arg {
+	PROFILER_ARG_U8,
+	PROFILER_ARG_S8,
+	PROFILER_ARG_U16,
+	PROFILER_ARG_S16,
+	PROFILER_ARG_U32,
+	PROFILER_ARG_S32,
+	PROFILER_ARG_STRING,
+	PROFILER_ARG_TIMESTAMP
+};
+
+
+/** @brief Buffer required for data that is sent with the event.
+ */
+struct log_event_buf {
+#ifdef CONFIG_PROFILER
```

Comment:
where is this CONFIG_PROFILER defined? This is a very generic kconfig to use and it is not defined in this PR, did you mean to rename it to something else?

Reply:
As agreed in previous comments all data related to profiler will be removed from event manager.